cypress/integration/supervision/dashboard/reassign-client.spec.js

```diff
@@ -0,0 +1,80 @@
+const createClient = () => {
+  return cy.fixture('client/minimal.json').then(client => {
+    cy.postToApi('/api/v1/clients', client)
+      .its('body')
+      .then(res => {
+        cy.wrap(res.caseRecNumber).as('courtReference');
+        cy.wrap(res.id).as('clientId');
+      });
+  });
+};
+
+const allocateTheClientToHWTeam = () => {
+  cy.request({
+    url: '/api/v1/teams',
+    headers: {
+      'accept': 'application/json',
+      'opg-bypass-membrane': 1
+    }
+  }).its('body')
+    .then(teams => {
+      for (let i in teams) {
+        if (teams[i].name === 'Health & Welfare - (Supervision)') {
+          cy.wrap(teams[i].id).as('hwTeamId');
+          break;
+        }
+      }
+    });
+
+  cy.get('@hwTeamId').then(hwTeamId => {
+    cy.get('@clientId').then(clientId => {
+      return cy.putToApi(`/api/v1/clients/${clientId}/edit/allocate`, `{"teamId":"${hwTeamId}"}`);
+    });
+  });
+}
+
+before(function setupAllocatedClient () {
+  cy.loginAs('Allocations User')
+    .then(createClient)
+    .then(allocateTheClientToHWTeam);
+});
+
+describe('Reassign client on the dashboard', { tags: ['@supervision', '@supervision-regression', '@xxx'] }, () => {
+  it(
+    'Given I\'m a Case Manager on Supervision and on the client dashboard page' +
+    'Then the Client Dashboard page loads as expected',
+    () => {
+      cy.loginAs('Lay User');
+      cy.visit('/supervision/#/dashboard');
+      cy.get('li[class="tab-container__tab TABS_CLIENTS"]').click();
+
+      cy.get('[data-role="main-filter-base-dropdown"]')
+       .find('select')
+       .select('AnotherTeam');
+
+      cy.get('[data-role="main-filter-team-dropdown"]')
+        .find('select')
+        .select('Health & Welfare - (Supervision)');
+
+      cy.get('[data-role="main-filter-apply-button"]').click({force: true});
```

Comment:
I'm a big fan of data attributes for this kind of thing. Worth documenting the preferred format now though, before the test suite gets too big. Can also be worth hiving them off into a file of constants to avoid typo issues by having linting catch them.

I've previously used a format like `data-action="entity.[activity.]action"` so` team.filter.apply` or `team.create`, if you include the interface element is can be brittle when you start changing links to buttons.